### ISSUE TITLE
[Epic] Create most popular SciKit Learn models

### ISSUE THREAD

**Issue by javabster** [IceKhan13/inferenza#5]
done = ✅ , in progress = 🏗 , in review = 👀 

 
- [[Support vector machines](https://scikit-learn.org/stable/modules/svm.html)]
    - SVC - @IceKhan13 
    - NuSVC - @IceKhan13 
    - LinearSVC - @IceKhan13 
    - SVR - @javabster 
    - NuSVR - @javabster 
    - LinearSVR - @javabster 
- Linear models:
    - Linear Regression ✅
    - Logistic Regression - @javabster 🏗
    - Linear Classifier
- Naive Bayes:
    - Gaussian NB 👀 
- Clustering:
    - kmeans ✅
- neighbours:
    - [[NearestNeighbors (KNN)](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.NearestNeighbors.html#sklearn.neighbors.NearestNeighbors)
- decision trees:
    - [[decision tree classifier](https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeClassifier.html#sklearn.tree.DecisionTreeClassifier)
    - [decision tree regressor](https://scikit-learn.org/stable/modules/generated/sklearn.tree.DecisionTreeRegressor.html#sklearn.tree.DecisionTreeRegressor)
- Gradient boosters:
    - [GradientBoostingClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html#sklearn.ensemble.GradientBoostingClassifier)
    - [GradientBoostingRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html#sklearn.ensemble.GradientBoostingRegressor)
